src/compiler/diagnosticMessages.json

```diff
@@ -2466,5 +2466,9 @@
     "A constructor cannot contain a 'super' call when its class extends 'null'": {
       "category": "Error",
       "code": 17005
+    },
+    "Left hand side of '**' cannot be a simple unary expression. Consider parenthesize the expression.": {

```

Comment:
"A unary expression starting with '{0}' must be parenthesized when on left-hand side of an exponentiation expression."

"The left-hand side of an exponentiation expression must be parenthesized when starting with '{0}'."

"The left-hand side of an exponentiation expression cannot start with '{0}'. Consider parenthesizing the expression."